frontend/src/components/MainSearch/MainSearch.js

```diff
@@ -49,18 +55,43 @@ class MainSearch extends Component {
             </p>
             <span className="search">
               <form className="search-form" onSubmit={this.handleSubmit}>
-                <input
-                  type="text"
-                  className="search-input"
-                  placeholder="Search by keywords"
-                  onChange={this.handleChange}
-                />
-                <button type="submit" className="search-button">
-                  <img
-                    src="https://img.icons8.com/ios/30/000000/search.png"
-                    alt="search"
-                  />
-                </button>
+                <div className="absoluteContainer">
+                  <div className="searchContainer">
+                    <input
+                      type="text"
+                      className="search-input"
+                      placeholder="Search by recipe name or ingredient"
+                      onChange={this.handleChange}
+                    />
+                    <button type="submit" className="search-button">
+                      <img
+                        src="https://img.icons8.com/ios/30/000000/search.png"
+                        alt="search"
+                      />
+                    </button>
+                  </div>
+                  <div className="buttonContainer">
+                    {!this.state.open ? (
+                      <div onClick={(event) => this.togglePanel(event)} className="button-box">
+                        <img
+                          src="https://img.icons8.com/ios/26/000000/chevron-down.png"
+                          alt="chevron button"
+                          className="chevron"
+                        />
+                      </div>
+                    ) : null}
+                  </div>
+                  {/* ternary operator shortcut (if/else) */}
+                  {this.state.open ? (
+                    <div className="content">
+                      <p> You can search recipe names or ingredients by keyword. Enter your keywords and click the search button or press enter to search. If you use multiple keywords, the search will return only results containing all of your keywords.</p>
+                      <div onClick={(event) => this.togglePanel(event)}>
+                        <p className="close">Collapse</p>
+                      </div>
+                    </div>
+
+                  ) : null}
```

Comment:
How about instead of having two separate ternaries, you just combine them?

Even better, it might be worth pulling this out into a separate function, like `renderExplainerPanel` instead of having a giant ternary in the middle of your render block. Then you can just use a real if statement.

Reply:
I think this change is worth making. Right now, it's pretty much:

```
if x
  do some stuff

if !x
  do some other stuff
```

Which should probably be more like:

```
if x
  do some stuff
else
  do some other stuff
```

Same thing with these ternaries: `x ? do some stuff : do some other stuff`. But either way, it's going to be more readable breaking it out into a separate function. 15-line ternaries are definitely not ideal.